keras_hub/src/models/gemma/gemma_attention.py

```diff
@@ -106,12 +108,14 @@ def _apply_rope(self, x, start_index):
         )
         return x
 
-    def _can_use_flash_attention(self):
-        if not has_flash_attention_support():
+    def _use_fused_attention_op(self):
+        if not fused_attention_op_available():
             return False
         if self.dropout > 0.0:
             return False
-        if self.logit_soft_cap is None:
+        if running_on_gpu() and not gpu_is_a100():
+            return False
+        if self.logit_soft_cap is None and running_on_gpu() and gpu_is_a100():
```

Comment:
I don't think we want to only run fused attention on an a100, what about h100? 5090? rtx pro 6000?

I think we are looking for a denylist instead of hardware that does not support the fused attetion op. t5 (maybe v100)